engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/management/RequestResponseProcessClient.scala

```diff
@@ -113,5 +118,21 @@ class HttpRequestResponseClient(managementUrl: String)(implicit backend: SttpBac
 
   override def close(): Unit = Await.result(backend.close(), Duration(10, TimeUnit.SECONDS))
 
+  private def handleUnitResponse(action: String)(response: Response[Either[String, String]]): Future[Unit] = (response.code, response.body) match {
```

Comment:
Can you move it to httputils, extracting also from flink client?

Reply:
Done, can you look right now?